components/form.js

```diff
@@ -453,7 +453,8 @@ function InputInner ({
         const isNumeric = /^[0-9]+$/.test(draft)
         const numericExpected = typeof field.value === 'number'
         helpers.setValue(isNumeric && numericExpected ? parseInt(draft) : draft)
-        onChange && onChange(formik, { target: { value: draft } })
+        // draft means change event is from local storage
+        onChange && onChange(formik, { target: { value: draft, draft: true } })
```

Comment:
It doesn't look like we are using `draft: true` anymore, are we?

Reply:
I removed it. Good catch.